soh/src/code/z_message_PAL.c

```diff
@@ -37,6 +37,10 @@ MessageTableEntry* sStaffMessageEntryTablePtr;
 
 char* _message_0xFFFC_nes;
 
+const int fastText = 5;
+const int slowText = 1;
+bool textSpeed = 1;
+
```

Comment:
If you follow my recommendations in https://discord.com/channels/808039310850130000/936124566764392510/964589497025200238, none of this will be necessary.